packages/cache/test/test-storage.js

```diff
@@ -47,21 +48,21 @@ test('makeChainStorageCoordinator with non-remote values', async t => {
   t.is(await cache('brandName', 'barbosa'), 'barbosa');
   t.deepEqual(Object.keys(storageNodeState), ['cache']);
   t.deepEqual(JSON.parse(storageNodeState.cache), {
-    '{"body":"\\"brandName\\"","slots":[]}': {
-      body: '{"generation":{"@qclass":"bigint","digits":"1"},"value":"barbosa"}',
+    '{"body":"#\\"brandName\\"","slots":[]}': {
```

Comment:
I'm not clear on why these were explicit instead of snapshot tests. Not worth more work now to change but it might have saved some had they been earlier.

Reply:
There were defintely tests that have a huge pile of encoded strings, which was obviously cut-and-pasted from a previous run, or form some manual tool that generated and serialized some original data. I left one of them in place (https://github.com/Agoric/agoric-sdk/blob/f243a2cb67bff4416069d1e558544a781d0d6a81/packages/vats/test/test-board-utils.js#L14) because it was only being *de*serialized by the test (the data represents what you might receive from an RPC call), and we're pretty confident that marshal knows how to deserialize old-style data, and it wasn't obvious where the data came from, so replacing it will involve writing a short tool to deserialize the old and re-serialize it with smallcaps.

In general, `t.snapshot` warns you when something changes unexpectedly, but runs the risk of not actually asserting that the data matches your original expectations. Anytime we add a `t.snapshot` test, the initial review should examine the `snapshot/NAME.md` file and confirm that it contains what the humans know it should contain. And of course, any PR which changes the contents should explain why it's ok for them to change.